Formula/czmq.rb

```diff
@@ -22,28 +21,26 @@ class Czmq < Formula
   end
 
   option :universal
+  option "with-drafts", "Build and install draft classes and methods"
```

Comment:
Can you explain a bit more about what this does and why you'd want to enable/disable it?